hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMDBCheckpointServlet.java

```diff
@@ -106,6 +139,35 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) {
       return;
     }
 
+    // Check ACL for dbCheckpoint only when global Ozone ACL is enable
+    if (om.getAclsEnabled()) {
+      final java.security.Principal userPrincipal = request.getUserPrincipal();
+      if (userPrincipal == null) {
+        // Fallback to checking login user if getUserPrincipal returns null.
+        // Note: In prod, a secure cluster with Kerberos should not hit this
+        // branch. This is mostly here for UT and dev testing.
+        final String remoteUser = request.getRemoteUser();
```

Comment:
null check for the remoteUser

Reply:
should be fine as the string isn't deferenced. just printed in the log.